apis_core/collections/signals.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from crum import get_current_request
+from apis_core.collections.models import SkosCollection, SkosCollectionContentObject
+from apis_core.history.models import VersionMixin
+from django.contrib import messages
+
+
+@receiver(post_save)
+def add_to_session_collection(
+    sender, instance, created, raw, using, update_fields, **kwargs
+):
+    """
+    Add a created object to all the SkosCollections that are listed
+    in the `session_collections` session variable.
+    """
+    request = get_current_request()
```

Comment:
ok, tracked it down.
I used the sample_project as is and in the sample project the crum middleware:
```python
MIDDLEWARE += ('crum.CurrentRequestUserMiddleware',)
```
is missing. And apparently the `get_current_request` function just returns False instead of raising an exception if thats missing.